src/Data/LLVM/BitCode/Bitstream.hs

```diff
@@ -19,15 +19,15 @@ module Data.LLVM.BitCode.Bitstream (
 import Data.LLVM.BitCode.BitString as BS
 import Data.LLVM.BitCode.GetBits
 
-import Control.Applicative ((<$>))
 import Control.Monad (unless,replicateM,guard)
 import Data.Bits (Bits, bitSizeMaybe, bit)
-import Data.Monoid (Monoid(..))
 import Data.Word (Word8,Word16,Word32)
+import qualified Data.Binary.Get as BG
 import qualified Data.ByteString as S
 import qualified Data.ByteString.Lazy as L
 import qualified Data.Map as Map
-import qualified Data.Serialize as C
+import Data.Bifunctor (bimap)
```

Comment:
`Data.Bifunctor` was introduced in `base-4.8`, so make sure to bump the lower version bounds in the `.cabal` file accordingly.